02-peer-protocol.md

```diff
@@ -745,6 +744,7 @@ affect the channel operation).
 The currently defined basic types are:
   - `option_static_remotekey` (bit 12)
   - `option_anchors` and `option_static_remotekey` (bits 22 and 12)
+  - `zero_fee_commitments` (bit 40)
```

Comment:
Why doesn't this channel type also have `option_static_remotekey`?

Reply:
> Why do you want to include the option_static_remotekey feature bit?

I thought that channel type is supposed to be all the features that a channel includes, so static remote key should be included here (because zero fee commitment channels will also use static remote keys).

It wasn't obvious to me that now that `option_static_remote_key` is assumed, it would also be assumed for channel types as well, but makes sense so that we can deprecate completely 👌

Reply:
> The issue with thinking that way is that we conflated channel types and lightning feature bits, so if we consider that 0-fee commitments use option_static_remotekey we end up being force to continue supporting the plain option_static_remotekey channel type, which we want to deprecate at some point (ideally, soon).

Yeah makes sense 👍  I was indeed conflating the two ideas.